packages/kit/src/vite/index.js

```diff
@@ -320,12 +319,26 @@ function kit() {
 			process.env.SVELTEKIT_SERVER_BUILD_COMPLETED = 'true';
 			log.info('Prerendering');
 
-			prerendered = await prerender({
-				config: svelte_config.kit,
-				client_out_dir: vite_config.build.outDir,
-				manifest_path,
-				log
-			});
+			const results_path = `${svelte_config.kit.outDir}/generated/prerendered.json`;
+
+			// do prerendering in a subprocess so any dangling stuff gets killed upon completion
+			spawnSync(
+				'node',
+				[
+					'./node_modules/@sveltejs/kit/dist/prerender.js',
```

Comment:
does this always work? Should either resolve the location or put it as bin in kit package

Reply:
It should work for npm and pnpm unless you've done something weird like install sveltekit globally or only installed it in the monorepo root. One benefit of the code as it is now is that it doesn't require us to expose it externally. We could add some validation to make sure that it does something sensible if the file is not there at least

Is there an easy way to resolve the package location? It sounds nice, but also like a big can of worms if I remember my experience with that from Vite. I think we need an extra package to do that and a lot of them are broken. I can't remember if we finally found a good one that works and is easy to use?

Reply:
> Is there an easy way to resolve the package location?

`createRequire(import.meta.url).resolve('@sveltejs/kit/package.json')` and some path manipulation to append dist/prerender.js if you don't want to expose it

Reply:
Or with just the local file path like https://github.com/sveltejs/vite-plugin-svelte/blob/308c879a4881c61f625d0dc072d65d85352c49a1/packages/vite-plugin-svelte/src/ui/inspector/plugin.ts#L16

Reply:
done. thanks! that was easy :smile: